pmml/step1_prepare/step1_2_preprocess_data.py

```diff
@@ -45,6 +45,14 @@
 
 # Extract number from reading level (e.g. 'LEVEL1' --> '1')
 storybooks_dataframe['reading_level'] = storybooks_dataframe['reading_level'].str.extract('(\\d+)')
+
+# Convert 'reading_level' into dummy variables
+storybooks_dataframe = pandas.concat(
+    [storybooks_dataframe.drop(columns=['reading_level']),  # Drop the original reading_level column
+     pandas.get_dummies(storybooks_dataframe['reading_level'], prefix='reading_level')],  # Add dummy columns
+    axis=1
+)
```

Comment:
@zakroum-hicham If you run your updated code, the CSV data output should be modified as well, but this PR didn't seem to contain any modifications to the CSV files?

https://github.com/elimu-ai/ml-storybook-reading-level?tab=readme-ov-file#run